packages/components/src/components/Table/styles/table.module.scss

```diff
@@ -0,0 +1,157 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+@import '../../../utilities/sizeToken.module.scss';
+@import '../../../utilities/focusStyle.module.scss';
+
+.table {
+    position: relative;
+    text-align: left;
+    border-collapse: collapse;
+
+    &[data-full-width='true'] {
+        width: 100%;
+    }
+
+    &[data-full-width='false'] {
+        width: auto;
+    }
+
+    &[data-layout='fixed'] {
+        table-layout: fixed;
+    }
+
+    &[data-layout='auto'] {
+        table-layout: auto;
+        [data-truncate='true'] {
+            max-width: 0;
+        }
+    }
+}
+
+caption {
+    padding: sizeToken(3) sizeToken(2);
+    text-align: left;
+    color: var(--text-color);
+}
+
+.header {
+    border-top: 1px solid var(--line-color);
+    border-bottom: 1px solid var(--line-color);
+
+    &[data-sticky='true'] {
+        position: sticky;
+        top: -1px;
+        z-index: 2;
+        background-color: var(--base-color);
+    }
+}
+
+.headerCell {
+    padding: sizeToken(3) sizeToken(2);
+    font-size: var(--body-size-small);
+    font-weight: 400;
+    line-height: sizeToken(4);
+    color: var(--text-color);
+
+    &[data-no-shrink='true'] {
+        width: 1%;
+        white-space: nowrap;
+    }
+}
+
+.sortButton {
+    display: inline-flex;
+    width: 100%;
+    text-align: inherit;
+    align-items: center;
+    gap: 0.25rem;
+
+    appearance: none;
+    background: none;
+    border: none;
+    padding: 0;
+    margin: 0;
+    font: inherit;
+    color: inherit;
+    cursor: pointer;
+
+    &:hover {
+        color: var(--text-color-hover);
+    }
+
+    &:focus-visible {
+        @include focus-outline;
+    }
+}
+
+.body {
+    padding: 0 sizeToken(6);
+
+    &[data-sticky-first-column='true'] {
```

Comment:
it doesn't work on the header at the moment. The first column header is still scrolling

Reply:
it's not meant to, only the content